web/packages/teleterm/src/deepLinks.ts

```diff
@@ -67,13 +70,23 @@ export type DeepURL = {
    * parseDeepLink decodes them so that other parts of the app don't have to deal with this.
    */
   username: string;
-  /**
-   * pathname is the path from the URL with the leading slash included, e.g. if the URL is
-   * "teleport://example.com/connect_my_computer", the pathname is "/connect_my_computer"
-   */
-  pathname: `/${Path}`;
 };
 
+/**
+ * pathname is the path from the URL with the leading slash included, e.g. if the URL is
+ * "teleport://example.com/connect_my_computer", the pathname is "/connect_my_computer"
+ */
+export type ConnectMyComputerDeepURL = BaseDeepURL & {
+  pathname: '/connect_my_computer';
+};
+
+export type AuthenticateWebDeviceDeepURL = BaseDeepURL & {
+  pathname: '/authenticate_web_device';
+  searchParams: { id: string; token: string };
```

Comment:
Are these URL parameters? Ie, `teleport://user@example.com/authenticate_web_device?id=a&token=b`?

Reply:
Yes. I discussed it with the team and it's easier to validate paths with query params so we went this route

Reply:
Alright by me, just confirming.  👍